ganttproject/src/net/sourceforge/ganttproject/search/SearchResultCellRenderer.java

```diff
@@ -0,0 +1,47 @@
+package net.sourceforge.ganttproject.search;
+
+import java.awt.Component;
+import java.awt.Font;
+
+import javax.swing.DefaultListCellRenderer;
+import javax.swing.JLabel;
+import javax.swing.JList;
+import javax.swing.ListCellRenderer;
+
+/**
+ * Created by Doug Kelly on 2/22/2018.
+ */
+
+public class SearchResultCellRenderer implements ListCellRenderer {
+
+    protected DefaultListCellRenderer defaultRenderer = new DefaultListCellRenderer();
+
+    @Override
+    public Component getListCellRendererComponent(JList jList, Object o, int index, boolean b, boolean b1) {
+        StringBuilder theText = new StringBuilder();
+        theText.append("<html>");
+
+        JLabel renderer = (JLabel) defaultRenderer.getListCellRendererComponent(jList, o, index,
+                b, b1);
+
+        if (o instanceof SearchResult) {
+            SearchResult searchResult = (SearchResult) o;
+            theText.append("<b>" + searchResult.getTypeOfResult()+ " #" + searchResult.getId() + "</b>: ");

```

Comment:
This should be internationalized, although admittedly it is not the easiest thing.

In general, we do i18n like this:
`GanttLanguage.getInstance().formatText(key, arg1, arg2)`. The implementation searches for the key value in appropriate resource bundle and [feeds the value and args](https://github.com/bardsoftware/ganttproject/blob/3ca29c5ba6facaf4b1fb8414bb831a8fcec0a096/ganttproject/src/net/sourceforge/ganttproject/language/GanttLanguage.java#L404) to `MessageFormat.format()`

I suggest extracting HTML markup of both labels into separate key-value pairs in the i18n resources and passing result type, id and label with highlighted match as arguments.  [Existing keys related to Search function](https://github.com/bardsoftware/ganttproject/blob/3ca29c5ba6facaf4b1fb8414bb831a8fcec0a096/ganttproject/data/resources/language/i18n.properties#L563) are prefixed with `search.` so please use the same prefix:

```
search.prompt.shortcut = <Ctrl+F>
search.result.line1 = <b>{0} #{1}</b>: {2}<br>
search.result.line2 = <b>{0}</b>: {1}
```